src/App.js

```diff
@@ -27,13 +45,20 @@ export default class App extends React.Component {
               <ViewSpace />
             </Route>
             <Route path="/survey">
-              <SurveyView />
+              <Survey />
```

Comment:
could you fix this?
* Survey is a component used in the View Spaces page. It's the jumbotron on the very bottom of the screen.
* SurveyView is the actual survey page.

you could rename them if you want but for now the functionality is broken for this